src/aat/resources/features/F-105 - Add Case-Assigned Users and Roles/common/F-105_User_Hemanth.td.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_guid_": "F-105_User_Hemanth",
+
+  "username": "befta.master.solicitor.benjamin@gmail.com",
+  "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_2_PWD]]"
```

Comment:
Confirm username value and the password variable used are correct after @MSancaktutar review of Benjamin & Becky references.

Reply:
Yes, correct.